superset-frontend/src/components/ReportModal/index.tsx

```diff
@@ -250,9 +251,16 @@ const ReportModal: FunctionComponent<ReportProps> = ({
         await dispatch(addReport(newReportValues as ReportObject));
         onHide();
       } catch (e) {
-        const parsedError = await getClientErrorObject(e);
-        const errorMessage = parsedError.message;
-        onReducerChange(ActionType.error, errorMessage);
+        const { error } = await getClientErrorObject(e);
+        if (error) {
```

Comment:
It seems the code changed from the time the issue was created.
The PR that does so is here: https://github.com/apache/superset/pull/17988

Looking at the PR, it seems the idea is to show the message in place, as opposed to use the toast.
Plus, looking at the change here, we're now assuming the error will always be a "report already exists" one, which is not correct.

I would suggest trying the following:
```js
const errorMessage = parsedError?.message || parsedError?.error;
```

If the error is not there, or if it's empty, we should use a default message I believe.
@rusackas thoughts here?

Reply:
@diegomedina248 @rusackas 

what is your correct expectation?

Reply:
For now, let's put the error at the bottom, like we do on the database connection modal. Consistency above all else!

<img width="673" alt="image" src="https://user-images.githubusercontent.com/812905/159062588-e564b1c2-bc2c-4d18-8002-595505843f98.png">